Tests/js/views/universaldiscovery/assets/ez-universaldiscoveryfinderexplorerlevelview-tests.js

```diff
@@ -379,13 +394,63 @@ YUI.add('ez-universaldiscoveryfinderexplorerlevelview-tests', function (Y) {
         },
     });
 
+    loadMoreItemsTest = new Y.Test.Case({
+        name: 'eZ Universal Discovery Finder Explorer load more items tests',
+
+        setUp: function () {
+            _fullLevelViewSetup(this);
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should update offset and fire locationSearch when scolling to unload items ": function () {
+            var container = this.view.get('container'),
+                locationSearchFired = false,
+                offset = this.view.get('offset');
+
+            this.view.on('*:locationSearch', Y.bind(function () {
+                Assert.isTrue(container.hasClass('is-loading'), 'Should have the loading icon');
+                locationSearchFired = true;
+            }), this);
+
+            container.setStyle('margin-top', 5000);
+            this.view.set('childCount', 999);
+
+            container.scrollInfo.fire('scrollDown');
+
+            Assert.areSame(this.view.get('offset'), offset + this.view.get('limit'), 'offset should be updated');
+            Assert.isTrue(locationSearchFired, 'locationSearch should have been fired');
+        },
+
+        "Should concat the new loaded items": function () {
+            var itemsLength = 10,
+                itemsArray = [];
+
+            for (var i = 0; i < itemsLength; i++) {
+                itemsArray.push(this.result);
+            }
+            this.view.set('items', itemsArray);
+            this.view.set('items', this.searchResult);
+
+            Assert.areSame(
+                this.view.get('items').length,
+                this.searchResultLength + itemsLength,
+                "items attribute should be filled with new item"
+            );
+        },
+    });
+
     Y.Test.Runner.setName("eZ Universal Discovery Finder Explorer Level View tests");
     Y.Test.Runner.add(renderTest);
     Y.Test.Runner.add(activeTest);
     Y.Test.Runner.add(navigateTest);
     Y.Test.Runner.add(searchResultChangeTest);
     Y.Test.Runner.add(scrollTest);
     Y.Test.Runner.add(removeHighlightTest);
+    Y.Test.Runner.add(loadMoreItemsTest);
 
-
-}, '', {requires: ['test', 'view', 'ez-universaldiscoveryfinderexplorerlevelview', 'node-screen', 'node-style', 'node-event-simulate']});
+}, '', {requires: ['test', 'view', 'ez-universaldiscoveryfinderexplorerlevelview',
+    'node-screen', 'node-style', 'node-scroll-info', 'node-event-simulate']});
```

Comment:
node-scroll-info is a dependency of ez-universaldiscoveryfinderexplorerlevelview not of this test so it should be added in ez-universaldiscoveryfinderexplorerlevelview.html in the module definition (and the indentation is weird)